# Supplementary Tables

- **Table-S1-Differentially-expressed-genes.xlsx**;
  List of differentially expressed probes(genes) based on ajusted-pvalue <= 0.05 and |log2(FC)| >= 1.0
  

- **Table-S2-pseudogenes-DE-clsuter-ID-annotations.xls**;
  List of pseudogenes differentially
  

- **Table-S3_qPCR_gene_expression_fold_change.pdf**;
Gene expression fold-change of study groups: Anti-tTG positive FDR, CeD, DC, Anti-tTG negative FDR.


- **Table-S4-data-for-FigureS2-venn-diagram.xlsx**;
  List genes up-regulated and down-regulated in FDR and CeD.
  
  
- **Table-S5a-GO-pval-0.05-DE.xlsx**;
  Gene ontology of differentially expressed genes (up and down)  for all comparisons.


- **Table-S5b-MSigDB_Consistently_down_in_FDR.xlsx**;
  MSigDB enrichment analysis based on genes which are down-regualted consistently in FDR in comparison to CeD and control.
  
   
- **Table-53c-Reactome-Down-in-FDR.xlsx**;
  Reactome pathway enrichment for down in FDR based on adjusted-pvalue <= 0.05 and without log2(FC) criteria.   


- **Table-S6-TranscriptionFactors-YY1-KLFs-NFKB-inhibitors.xlsx**;


- **Table-S7a-pattern-analysis.xlsx**;
  List of probes(genes) for each gene expression pattern.


- **Table-S7b-GO_pattern.xlsx**;
  Gene ontology based on genes observed in each expression pattern. 








